README.rst

```diff
@@ -43,10 +43,22 @@ To enhance this BO4E implementation and contribute to this project check out the
 
 .. code-block:: Shell
 
-   tox -e dev 
-   
+   tox -e dev
+
 The created venv should be located somewhere around .tox/dev/Scripts.
 
+Regular Expression for Enumerations
+-----------------------------------
+
+If you want to add a new enumeration from the `BO4E Standard`_ then you can use the following regular expression:
```

Comment:
I wouldn't refer to the bo4e.de website as "standard" but as just what it is: the only source of truth we have: a plain html table to copy paste from.

just for clarity so that the reader knows where the input actually comes from.

Reply:
I changed the name to BO4E website